modules/es-extensions/store/extensions/app/greg_store/pages/greg-landing.jag

```diff
@@ -22,7 +22,9 @@ require('/modules/store.js').exec(function(ctx) {
 
 	var log = new Log();
 	log.debug("Printing user ############" + stringify(user));
```

Comment:
Please initialise the log object and wrap the log with isDebugEnabled if condition.